specreduce/extract.py

```diff
@@ -106,11 +106,19 @@ class BoxcarExtract(SpecreduceOperation):
     spec : `~specutils.Spectrum1D`
         The extracted 1d spectrum expressed in DN and pixel units
     """
-
+    image: NDData
+    trace_object: Trace
+    width: float = 5
+    disp_axis: int = 1
+    crossdisp_axis: int = 0
```

Comment:
1. Should these defaults be noted in the `BoxcarExtract` docstring? They are in `HorneExtract`.
2. The API example in the `BoxcarExtract` docstring needs an update to match the new API introduced here.

Reply:
good catch, done!